container.go

```diff
@@ -336,8 +334,7 @@ func (container *container) Start() error {
 	return nil
 }
 
-// Shutdown requests a container shutdown, if IsPending() on the error returned is true,
```

Comment:
This `IsPending` is different than PendingUpdates. It is about the HCS doing the shutdown/terminate on a different thread as an async task.

Reply:
Ah, yes, you're right. I was over-zealous on removing. Reinstated